libs/SalesforceSDK/res/values/sf__strings.xml

```diff
@@ -8,6 +8,7 @@
 	<string name="sf__invalid_server_url">Please type in a valid URL starting with https://</string>
 	<string name="sf__access_token_revoked">Your access token has been revoked by the administrator. You will now be logged out.</string>
     <string name="sf__managed_app_error">Authentication only allowed from managed device.</string>
+	<string name="sf__jwt_authentication_error">JWT authentication error. Please try again.</string>
```

Comment:
Actually, you wouldn't modify this file. You would override the value for this key in your app's `strings.xml`.